assets/helpers/csrf/csrfActions.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+// ----- Types ----- //
+
+export type Action = { type: 'SET_TOKEN', name: string };
+
+
+// ----- Actions Creators ----- //
+
+export function setToken(name: string): Action {
+  return { type: 'SET_TOKEN', name };
+}
+
+export default setToken;
+
```

Comment:
Will this action ever get used? If not, we may not need it; this state will just remain static client-side after being hydrated from the server.